common/modeling.py

```diff
@@ -53,7 +53,10 @@ class Usage(pg.Object):
 class LMSamplingResult(lf.LMSamplingResult):
   """LMSamplingResult with usage information."""
 
-  usage: Usage | None = None
+  usage: Annotated[
+      lf.LMSamplingUsage,
+      'Usage information. Currently only OpenAI models are supported.',
```

Comment:
This sounds a bit misleading, so we probably should not make the change here.

Reply:
Langfun introduced this change: https://github.com/google/langfun/commit/05d606297c567fd591dfa4d9bc5dbe8f7533b433#diff-1cccb97692d44736a199e067f263aaacee2dc72af3f77eee9e875d3a76a2489eL108